recipe/meta.yaml

```diff
@@ -10,29 +10,29 @@ source:
 
 build:
   skip: True  # [py<37]
-  number: 0
+  number: 1
   script:
-    - python -c '__import__("os").unlink("pyproject.toml")'
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
     - wheel
-    - poetry
+    - poetry-core >=1.0.0
   run:
     - python
-    - typing_extensions >=4.4  # [py<310]
+    - typing-extensions >=4.7.1  # [py<38]
+    - typing-extensions >=4.9    # [py>=38]
 
 test:
   imports:
     - graphql
   source_files:
     - tests
   requires:
-    - pytest >=6.2,<7
-    - pytest-benchmark >=3.4,<4
+    - pytest >=7.4,<8.1 
+    - pytest-benchmark >=4.0.0
```

Comment:
`pytest >=6.2,<7` and  `pytest-benchmark >=3.4,<4` are correct for **v3.2.3**, see https://github.com/graphql-python/graphql-core/blob/v3.2.3/pyproject.toml#L58-L58C17. 
I think you were looking at the **main** branch instead of the tag **v3.2.3**